docs/sources/getting-started/getting-started.md

```diff
@@ -10,9 +10,9 @@ weight = 200
 
 This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "_index.md" >}}).
 
-## Step 1: Install Grafana
+## Step 1: Install Grafana to self host or get a Free Grafana Cloud Account 
 
-Grafana can be installed on many different operating systems. For a list of minimum hardware and software requirements, as well as instructions on installing Grafana, refer to [Install Grafana]({{< relref "../installation/_index.md" >}}).
+Grafana offers a [free account with Grafana Cloud](https://grafana.com/signup/cloud/connect-account?pg=gsdocs) (10K metrics, 50gb logs and dashboards) to make starting even easier and faster. Alternatively, Grafana can be installed on many different operating systems. For a list of minimum hardware and software requirements, as well as instructions on installing Grafana, refer to [Install Grafana]({{< relref "../installation/_index.md" >}}).
```

Comment:
+1 to not include plan-specific details in a Getting started document.

This causes a maintenance issue where those numbers are subject to change without timely cross-team knowledge of such an information dependency, and as a result user become confused when they sees specifics in one area that do match specifics in another area.

Reply:
We can kill the reference to 10K metrics, 50gb logs and dashboards, but I want the Grafana Cloud with link to a signup in there.

Reply:
Agree with @osg-grafana. This content should really be in the topic "What is Grafana?" but I based my review keeping in mind @sfingerhut's past experiences.